binding/Binding/Definitions.cs

```diff
@@ -97,7 +97,123 @@ public SKColor WithAlpha (byte alpha)
 		public byte Green => (byte)((color >> 8) & 0xff);
 		public byte Blue => (byte)((color) & 0xff);
 
-		public override string ToString ()
+		public static SKColor FromHSV (float hue, float saturation, float value)
+		{
+			if (saturation < 0 || saturation > 1)
+				throw new ArgumentException ("saturation");
+			if (value < 0 || value > 1)
+				throw new ArgumentException ("value");
+
+			byte s = SkUnitScalarClampToByte (saturation);
+			byte v = SkUnitScalarClampToByte (value);
+
+			if (0 == s) { // shade of gray
+				return new SKColor (v, v, v);
+			}
+
+			int hx = (hue < 0 || hue >= 360.0f) ? 0 : SkScalarToFixed (hue / 60.0f);
+			int f = hx & 0xFFFF;
+
+			uint v_scale = SkAlpha255To256 (v);
+			uint p = SkAlphaMul (255u - s, v_scale);
+			uint q = SkAlphaMul (255u - (uint)(s * f >> 16), v_scale);
+			uint t = SkAlphaMul (255u - (uint)(s * ((1 << 16) - f) >> 16), v_scale);
+
+			uint r, g, b;
+
+			switch (hx >> 16) {
+			case 0: r = v; g = t; b = p; break;
+			case 1: r = q; g = v; b = p; break;
+			case 2: r = p; g = v; b = t; break;
+			case 3: r = p; g = q; b = v; break;
+			case 4: r = t; g = p; b = v; break;
+			default: r = v; g = p; b = q; break;
+			}
+			return new SKColor ((byte)r, (byte)g, (byte)b);
+		}
+
+		public static void ToHSV(SKColor color, out float hue, out float saturation, out float value)
```

Comment:
It is not clear why we would like to have this static method, as it requires an SKColor to be there.

I rather just keep the instance method.